.mocharc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  file: ['./test/mocha-utils.js'],
+  spec: 'test/*.mocha.spec.js',
+  reporter: 'dot',
+}
```

Comment:
match existing coding style

```suggestion
};
```